src/components/dataset/schemaCreation/DatasetSchemaCreationView.tsx

```diff
@@ -75,93 +90,139 @@ interface TabConfig {
 }
 
 const DatasetSchemaCreationView = withStyles(styles)(({ classes }: IProps) => {
-  const [value, setValue] = React.useState(0);
-  const handleChange = (_event: any, newValue: any) => {
-    setValue(newValue);
-  };
+  const [currentTab, setCurrentTab] = React.useState(0);
+  const changeTab = (_event: any, newCurrentTab: any) => setCurrentTab(newCurrentTab);
+
+  const formMethods = useForm({
+    mode: 'onTouched',
+    reValidateMode: 'onChange',
+    defaultValues: {
+      name: '',
+      description: '',
+      terraProject: '',
+      enableSecureMonitoring: 'true',
+      cloudPlatform: 'gcp',
+      region: '',
+      stewards: [],
+      custodians: [],
+    },
+  });
+
+  const { handleSubmit } = formMethods;
 
   const tabs: TabConfig[] = [
     {
       description: 'Provide dataset information',
-      content: <div>Step 1!</div>,
+      content: <DatasetSchemaInformationView />,
     },
     {
       description: 'Build a schema and create dataset',
       content: <div>Step 2!</div>,
     },
   ];
 
+  const onSubmit = (data: any) => {
```

Comment:
Is there a reason to have this function before it does anything? It looks like it is feature flagged, but my general rule tends to be waiting to add the method until implementation time to avoid todo code ending in production.

Reply:
Sure, if that is team agreement, that's probably okay.